opbeat/base.py

```diff
@@ -160,6 +159,15 @@ def __init__(self, organization_id=None, app_id=None, secret_token=None,
             msg = 'Missing configuration for client. Please see documentation.'
             self.logger.info(msg)
 
+        self.is_send_disabled = (
+            os.environ.get('OPBEAT_DISABLE_SEND', '').lower() == 'true'
```

Comment:
I think '1' is also a common way to set ENV vars. Could be `... in ('1', 'true')` ?